src/rachanaGJun20/CoadingExam/CodingExam10.java

```diff
@@ -0,0 +1,40 @@
+/*Coding test 10 :
+
+Print minimum rotation required to compare both strings equality.
+
+Example 1  : 
+String input1 = "abcdef";
+String input2 = "defabc";
+output : 3
+
+Example 2  : 
+String input1 = "adbcdef"; -> push right
+fadbcde
+efadbcd
+defadbc
+String input2 = "defadbc";
+output : 3*/
+package rachanaGJun20.CoadingExam;
+
+public class CodingExam10 {
+	
+	static void compareString() {
+		String input1 = "abcdef";
+		String input2 = "defabc";
+		int iterationCount = 0;
+		if (input1.length() == input2.length()) {
+			while (!input1.equals(input2)) {
```

Comment:
what if String can't be made equals even after all possible rotation ?? this will run infinitely ...

you can handled it with one more condition.
while(!input1.equals(input2) && iterationCount< input.length)